exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -122,10 +122,10 @@
   String HTTP_SESSION_MEMORY_RESERVATION = "drill.exec.http.session.memory.reservation";
   String HTTP_SESSION_MEMORY_MAXIMUM = "drill.exec.http.session.memory.maximum";
   String HTTP_SESSION_MAX_IDLE_SECS = "drill.exec.http.session_max_idle_secs";
-  String HTTP_KEYSTORE_PATH = "javax.net.ssl.keyStore";
```

Comment:
Hmm, not entirely convinced, but I'm fine as long as you have provided backward compatibility for existing users.